chapters/part3/hyperparameter-optimization-with-optuna.md

```diff
@@ -0,0 +1,426 @@
+---
+title: 'Hyperparameter Optimization with Optuna'
+description: "This chapter gives a basic tutorial for optimizing the hyperparameters of your model."
+type: chapter
```

Comment:
You can put an "author" key in here, to have you get credit for writing this (won't render correctly until #134 is merged, though).

Reply:
Added in 1119a64. Thank you.